config/jest/test.config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+    rootDir: "../../",
+    transform: {
+        "^.+\\.jsx?$": "<rootDir>/config/jest/test.transform.js",
+    },
+    testEnvironment: "jest-environment-jsdom",
+    globals: {
+        SNAPSHOT_INLINE_APHRODITE: true,
+    },
+    testMatch: ["<rootDir>/packages/**/*.test.js"],
+    setupTestFrameworkScriptFile: "<rootDir>/config/jest/test-setup.js",
+    moduleNameMapper: {
+        "^@khanacademy/(.*)$":
+            "<rootDir>/node_modules/@khanacademy/$1/index.js",
```

Comment:
This change is what's causing the increase in test coverage.  It's cause the normal `jest` tests to import the source `index.js` file instead of `dist/index.js` which we aren't collecting coverage on.

Reply:
Good. I wanted to add test coverage of the regular `index.js` (and did for some of our packages), but adding it for all is great. Of course, we should have unit tests for them to verify the public API is what we expect, as well as the vite things, but baby steps.